solutions/instances/README.md

```diff
@@ -1,16 +1,16 @@
-# IBM Observability Instances Deployable Architecture
+# IBM Cloud observability instances deployable architecture
 
-This architecture creates the observability instances on IBM Cloud and supports provisioning the following resources:
+This deployable architecture creates observability instances in IBM Cloud and supports provisioning the following resources:
 
-- A resource group, if one is not passed in.
-- A Log Analysis instance.
-- A Cloud Monitoring instance.
-- Creates a Cloud Object Storage (COS) instance or supports using an existing one.
-- Supports creating KMS root keys in an existing KMS instance or using existing keys if creating new buckets.
-- Creates a KMS encrypted COS bucket that is required to store archived logs or using an existing bucket.
-- Creates a KMS encrypted COS bucket for setting up Activity Tracker event routing or using an existing bucket.
-- Creates the Activity Tracker event route to a COS target.
+* A resource group, if one is not already passed in.
+* A Log Analysis instance.
+* A Cloud Monitoring instance.
+* A Cloud Object Storage instance, if one exists, or creates an instance.
```

Comment:
Just to clarify, if Cloud Object Storage exists then it does not provision. This deployable architecture has the capability to use the existing one. With this explanation, please check if rewording is required.

Reply:
We can update the bullet point to use the following text:

"A Cloud Object Storage instance, if one does not exist."

Reply:
sure, that works.